taxi/views.py

```diff
@@ -44,6 +45,26 @@
     model = Car
 
 
+class CarCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:car_list")
+    template = "taxi/car_form.html"
+
+
+class CarUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Car
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:car_list")
+    template = "taxi/car_form.html"
```

Comment:
The attribute `template` should be `template_name` to correctly specify the template used by this view.